portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.data;
+
+public class Comment{
+    //personal data
+    private String firstName, lastName, email, phone;
+    //job related data: optional
+    private boolean jobRelated=false;
```

Comment:
Is this field necessary? We could know this based on `jobTitle` being null or not?

Reply:
removed field